code/game/gamemodes/clock_cult/clock_structures/tinkerers_daemon.dm

```diff
@@ -14,8 +14,8 @@
 	debris = list(/obj/item/clockwork/alloy_shards/medium = 1, \
 	/obj/item/clockwork/alloy_shards/small = 6, \
 	/obj/item/clockwork/component/replicant_alloy/replication_plate = 1)
-	var/image/daemon_glow
-	var/image/component_glow
+	var/static/mutable_appearance/daemon_glow = mutable_appearance('icons/obj/clockwork_objects.dmi', "tinkerglow")
```

Comment:
uh, does this work if you have multiple daemons active and set to different things?